modules/game.feature.control.control.d.ts

```diff
@@ -13,13 +13,17 @@ declare global {
         this: this,
         destVec: Vec2,
         lastRelativeVel: number,
-        ignoreLastDir?: number | null,
+        ignoreLastDir?: Optional<number>,
       ): number;
       autoThrown(this: this): boolean;
       menuScrollUp(this: this): boolean;
       menuScrollDown(this: this): boolean;
       getMouseX(this: this): number;
       getMouseY(this: this): number;
+      menuHotkeyHelp(this: this): boolean;
+      menuHotkeyHelp2(this: this): boolean;
+      menuHotkeyHelp3(this: this): boolean;
+      menuBack(this: this): boolean;
```

Comment:
Additionally, `moveDir` must be below `autoThrown`, but I couldn't include it in the suggestion because the line containing the function name is outside of the hunk in this diff
```suggestion
      getMouseX(this: this): number;
      getMouseY(this: this): number;
      autoThrown(this: this): boolean;
      menuBack(this: this): boolean;
      menuHotkeyHelp(this: this): boolean;
      menuHotkeyHelp2(this: this): boolean;
      menuHotkeyHelp3(this: this): boolean;
      menuScrollUp(this: this): boolean;
      menuScrollDown(this: this): boolean;
```